atomic_reactor/plugins/exit_push_floating_tags.py

```diff
@@ -42,11 +42,9 @@ def add_floating_tags(self, session, manifest_list_data, floating_images):
             self.log.debug("storing %s as %s", target_repo, image.tag)
             self.manifest_util.store_manifest_in_repository(session, manifest, list_type,
                                                             target_repo, target_repo, ref=image.tag)
-        # And store the manifest list in the push_conf
-        push_conf_registry = self.workflow.push_conf.add_docker_registry(session.registry,
-                                                                         insecure=session.insecure)
+
         for image in floating_images:
-            push_conf_registry.digests[image.tag] = manifest_digest
+            self.workflow.conf.registry['digests'][image.tag] = manifest_digest
         registry_image = get_unique_images(self.workflow)[0]
 
         return registry_image.get_repo(explicit_namespace=False), manifest_digest
```

Comment:
it would be probably more useful if this returned registry_image.to_str and whole plugin would return dict with that as key and digests as values

Reply:
I have currently just changed it to query the registry for digest